### PR TITLE
new acrolinx files for integration

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** new acrolinx files for integration
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-654071](https://oktainc.atlassian.net/browse/OKTA-654071)

### CODE REVIEW

**Review by paulwallace-okta — DISMISSED** [review]
🚀

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,2 @@
-*    @susanharper-okta @kimmaida-okta @janethu-okta @grahamsmith-okta @barbaravo-okta @MarkCowan-okta @emikhasyak-okta @paulwallace-okta
+*    @susanharper-okta @kimmaida-okta @janethu-okta @grahamsmith-okta @barbaravo-okta @emikhasyak-okta @paulwallace-okta
+/acrolinx/global-config.edn @janethu-okta @susanharper-okta
```
Comment (paulwallace-okta) [inline .github/CODEOWNERS:2]: `/.github/acrolinx/global-config.edn` would be a complete path? `/` is the root of the repo, everything after that is from root.

Comment (susanharper-okta) [inline .github/CODEOWNERS:2]:
```suggestion
/.github/acrolinx/global-config.edn @janethu-okta @susanharper-okta
```

**Review by paulwallace-okta — APPROVED** [review]
🚀

**Pull request merged**